docs/usage/SYNC_OPTIONS.md

```diff
@@ -0,0 +1,62 @@
+# Sync Options
+
+#### `errorCallBack`
+a callback that is called whenever an error event occurs during the sync process.
+
+#### `warningCallBack` 
+a callback that is called whenever a warning event occurs during the sync process.
+
+#### `beforeUpdateCallback`
+a filter function which can be applied on a generated list of update actions. It allows the user to intercept 
+**_update_** actions just before they are sent to CTP API.
+
+#### `beforeCreateCallback`
+a filter function which can be applied on a draft before a request to create it on CTP is issued. It allows the 
+user to intercept **_create_** requests modify the draft before the create request is sent to CTP API.
+
+#### `batchSize`
+a number that could be used to set the batch size with which resources are fetched and processed with,
+as resources are obtained from the target CTP project in batches for better performance. The algorithm accumulates up to
+`batchSize` resources from the input list, then fetches the corresponding resources from the target CTP project
+in a single request. Playing with this option can slightly improve or reduce processing speed.
+
+#### `syncFilter` (Only for Product Sync Options)
+ represents either a blacklist or a whitelist for filtering certain update action groups. 
+  - __Blacklisting__ an update action group means that everything in products will be synced except for any group 
+  in the blacklist. A typical use case it to blacklist prices when syncing products, so as to sync everything in products
+  except prices. 
+  
+  - __Whitelisting__ an update action group means that the groups in this whitelist will be the *only* group synced in 
+  products. One use case could be to whitelist prices when syncing products, so as to only sync prices in products and
+  nothing else.
+  
+  - The list of action groups allowed to be blacklist or whitelisted on products can be found [here](/src/main/java/com/commercetools/sync/products/ActionGroup.java). 
+
+#### `ensureChannels` (Only for Product and Inventory Sync Options)
+a flag which represents a strategy to handle syncing inventory entries with missing channels.
```

Comment:
```suggestion
a flag, which represents a strategy to handle syncing inventory entries with missing channels.
```

Reply:
I don't this comma is grammatically correct: https://en.oxforddictionaries.com/punctuation/comma